src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import (
+    ImportImageParameters,
+    ImportSource,
+    ImportMode
+)
+from ._utils import (
+    validate_managed_registry,
+    get_registry_from_name
+)
+
+SOURCE_REGISTRY_MISING = "Please specify the source container registry name, login server or resource ID: "
+IMPORT_NOT_SUPPORTED = "Imports are only supported for managed registries."
+INVALID_SOURCE_IMAGE = "Please specify source image in the form of '[registry.azurecr.io/]repository:tag' or " \
```

Comment:
`'[registry.azurecr.io/]repository[:tag]'`, also `the` source image?
Could you check if `tag` is optional or not?

Reply:
I recall the server side attaches a `latest` if tag is empty, could you validate?

Reply:
Why is IMPORT_NOT_SUPPORTED a client side check. I think we should move this to the server and throw if called on a classic registry. - @nathana1

Reply:
Wow really? I am so curious. Can you explain more on what do you mean by moving to the server? As I have checked all the modules in the commands including build-task, custom, repository and web hook. All of them seem to use the validate_managed_registries and have defined a static constant to be XXX_NOT_SUPPORTED